hyasinsight.json

```diff
@@ -0,0 +1,3793 @@
+                    "column_order": 1,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "width": 12,
+                "height": 5,
+                "view": "hyasinsight_view.display_view",
+                "title": "Domain Lookup"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "lookup current whois domain",
+            "identifier": "lookup_current_whois_domain",
+            "description": "Perform this action to get the Whois current Domain Lookup Data for Hyas Insight",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "domain": {
+                    "description": "Domain to get Lookup Data for Hyas Insight",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "domain"
+                    ],
+                    "order": 0,
+                    "name": "domain"
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.domain",
+                    "data_type": "string",
+                    "contains": [
+                        "domain"
+                    ],
+                    "example_values": [
+                        "google.com"
+                    ]
+                },
+                {
+                    "data_path": "action_result.*.datetime",
+                    "data_type": "string",
+                    "example_values": [
+                        "2022-03-14T11:05:14Z"
+                    ]
+                },
+                {
+                    "data_path": "action_result.*.domain",
+                    "data_type": "string",
+                    "contains": [
+                        "domain"
+                    ],
+                    "example_values": [
+                        "hyas.com"
+                    ]
+                },
+                {
+                    "data_path": "action_result.*.abuse_emails",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "example_values": [
+                        "abuse@godaddy.com"
```

Comment:
Use dummy values instead of real names, Check for similar values in all the data paths.